iib/workers/tasks/opm_operations.py

```diff
@@ -31,6 +31,7 @@
 )
 
 log = logging.getLogger(__name__)
+worker_config = get_worker_config()
```

Comment:
Do we need it as global variable?

We usually have it like this:
```
    conf = get_worker_config()
    return _serve_cmd_at_port(
        serve_cmd, cwd, port, conf['iib_grpc_max_tries'], conf['iib_grpc_init_wait_time']
    )
```

Reply:
changed it. Also, moved the code to a function to avoid duplication across endpoints.